src/main.rs

```diff
@@ -88,11 +88,33 @@ impl Options {
 }
 
 fn pg_client(options: &Options) -> Result<Client, Box<dyn Error>> {
-    let connector = TlsConnector::builder()
-        .danger_accept_invalid_certs(true)
-        .build()?;
-    let connector = MakeTlsConnector::new(connector);
-    Ok(Client::connect(&options.database_url, connector)?)
+    mod danger {
+        pub struct NoCertificateVerification {}
+
+        impl rustls::client::ServerCertVerifier for NoCertificateVerification {
+            fn verify_server_cert(
+                &self,
+                _end_entity: &rustls::Certificate,
+                _intermediates: &[rustls::Certificate],
+                _server_name: &rustls::ServerName,
+                _scts: &mut dyn Iterator<Item = &[u8]>,
+                _ocsp_response: &[u8],
+                _now: std::time::SystemTime,
+            ) -> Result<rustls::client::ServerCertVerified, rustls::Error> {
+                Ok(rustls::client::ServerCertVerified::assertion())
+            }
+        }
+    }
+
+    let mut config = rustls::ClientConfig::builder()
+        .with_safe_defaults()
+        .with_root_certificates(rustls::RootCertStore::empty())
+        .with_no_client_auth();
+    config
+        .dangerous()
+        .set_certificate_verifier(Arc::new(danger::NoCertificateVerification {}));
+    let tls = tokio_postgres_rustls::MakeRustlsConnect::new(config);
+    Ok(Client::connect(&options.database_url, tls)?)
```

Comment:
After the last changes, I remembered [Clar's PR to do something similar](https://github.com/Blissfully/pg_parcel/pull/18). They had also added a command-line flag, so that it wasn't dangerous by default. What do you think about incorporating that flag in here too?

Reply:
addressed

Reply:
The earlier line `.with_root_certificates(rustls::RootCertStore::empty())` combined with _not_ allowing invalid certificates might be a bit problematic, as in I guess nothing will validate? 😅

Maybe that can be a problem for another day. @jelder Thoughts?

Reply:
`with_root_certificates` is about client certs, according to the docs:
```
Choose how to verify client certificates
```
we do not use nor require client certificates so I guess it's still OK.